templates/theme/styles/base.less

```diff
@@ -277,6 +279,7 @@ atom-text-editor .search-results .marker.current-result .region,
     font-style: italic;
   }
 
+  &.heading,
   &.heading .punctuation.definition.heading {
```

Comment:
What's actually an example for `&.heading .punctuation.definition.heading`? I can't find where that gets used. Maybe we can remove it and just have `&.heading`.